lib/shimmer.js

```diff
@@ -339,23 +340,25 @@ const shimmer = (module.exports = {
       }
 
       const shim = shims.createShimFromType(core.type, agent, mojule, mojule)
-      applyDebugState(shim, core)
+      applyDebugState(shim, core, false)
       _firstPartyInstrumentation(agent, filePath, shim, uninstrumented, mojule)
     }
   },
-  // TODO: remove the following methods once test-utilities have been udpated and released
+  // TODO: remove the following methods once test-utilities have been updated and released
   // with the new registerHooks and removeHooks methods
   patchModule(agent) {
     shimmer.registerHooks(agent)
   },
   unpatchModule() {
     shimmer.removeHooks()
   },
-
   registerHooks(agent) {
     this._ritm = new Hook(pkgsToHook, function onHook(exports, name, basedir) {
       return _postLoad(agent, exports, name, basedir)
     })
+    this._iitm = new IitmHook(pkgsToHook, function onESMHook(exports, name, basedir) {
+      return _postLoad(agent, exports, name, basedir, true)
+    })
   },
   removeHooks() {
```

Comment:
Shouldn't we also clear out `_iitm` / unhook like we do for `_ritm`?

Reply:
hmm i _did_, must've lost this when i rebased